Frontend/src/app/maps/maps/maps.component.ts

```diff
@@ -19,7 +19,7 @@ import { TrackingPoint } from '../../shared/tracking-point';
     styleUrls: ['./maps.component.css']
 })
 export class MapsComponent implements OnInit {
-    @ViewChild('map') mapElement: ElementRef;
+    @ViewChild('map',{static: false}) mapElement: ElementRef;
```

Comment:
why is this needed?

Reply:
In Angular's post, in version 8, we are migrating all applications and libraries to explicitly specify the resolution strategy for @ViewChild and @ContentChild queries.